<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsResults</class>
 <widget class="QDialog" name="OptionsResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>I3S Pattern: Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>All or best individual match</string>
   </property>
   <widget class="QRadioButton" name="radShowAllMatches">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show all matches of an individual</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radShowBestMatch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show best match of an individual only</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>File or folder in results window</string>
   </property>
   <widget class="QRadioButton" name="radShowFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show file name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radShowFolderName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show folder name</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Number of batch results</string>
   </property>
   <widget class="QLineEdit" name="txtNumBatchResults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Show results</string>
   </property>
   <widget class="QRadioButton" name="radShowTop50">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show top 50</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radShowAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show all</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Logging results</string>
   </property>
   <widget class="QRadioButton" name="radLoggingOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logging off</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radLoggingOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logging on</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
